### PR TITLE
Minor fixes and notification updates

### PR DESCRIPTION
**Pull request by maciaszczykm**
### What this PR does / why we need it
See commits.

Feel free to post suggestions related to the notification messages. It's an attempt to be more consistent but I think we still need to meet and create some patterns for the messages and them use them.

### Which issue(s) this PR fixes
A step towards https://github.com/kubermatic/dashboard/issues/4042.

https://github.com/kubermatic/dashboard/issues/4097 will cover the refesh part as it is a bit bigger change.

### Release note
```release-note
NONE
```

### CODE REVIEW

File: src/app/backup/list/restore/component.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export class RestoreListComponent implements OnInit, OnDestroy {
         )
       )
       .subscribe(_ => {
-        this._notificationService.success(`Successfully deleted restore object ${restore.name}`);
+        this._notificationService.success(`Deleted restore object ${restore.name}`);
```
Comment (kgroschoff) [inline src/app/backup/list/restore/component.ts:130]: For e.g. automatic backups we use `Deleted the ${backup.name} automatic backup` (so basically Action-Name-Resource)
and here we use `Deleted restore object ${restore.name}` (Action-Resource-Name)

I guess we should think about one order in general, shouldn't we?
[Haven't checked on all yet, but was the first one where I noticed it.]

Comment (kgroschoff) [reply src/app/backup/list/restore/component.ts:130]: cc @cschieder

Comment (maciaszczykm) [reply src/app/backup/list/restore/component.ts:130]: IMO the first form is better.

Comment (cschieder) [reply src/app/backup/list/restore/component.ts:130]: Iirc we use action-name-resource basically everywhere, like kristin said. another example: "Delete silly-clustername cluster" Imo it reads well.

**Comment by maciaszczykm** [comment]
@kgroschoff @floreks PTAL

**Pull request merged**